server/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -900,6 +907,21 @@ protected final RefreshResponse refresh(String... indices) {
         return actionGet;
     }
 
+    /**
+     * Returns <code>true</code> iff the given index exists otherwise <code>false</code>
+     */
+    protected static boolean indexExists(String index) {
+        return indexExists(index, client());
+    }
+
+    /**
+     * Returns <code>true</code> iff the given index exists otherwise <code>false</code>
+     */
+    public static boolean indexExists(String index, Client client) {
```

Comment:
Same here. Could we instead use SQL on information schema tables or something like that?